cmd/goal/clerk.go

```diff
@@ -161,6 +162,7 @@ func init() {
 	simulateCmd.Flags().BoolVar(&simulateAllowMoreLogging, "allow-more-logging", false, "Lift the limits on log opcode during simulation")
 	simulateCmd.Flags().BoolVar(&simulateAllowExtraOpcodeBudget, "allow-extra-opcode-budget", false, "Apply max extra opcode budget for apps per transaction group (default 320000) during simulation")
 	simulateCmd.Flags().Uint64Var(&simulateExtraOpcodeBudget, "extra-opcode-budget", 0, "Apply extra opcode budget for apps per transaction group during simulation")
+	simulateCmd.Flags().BoolVar(&simulateReturnPCTrace, "with-program-counter", false, "Return PC trace of app calls during simulation")
```

Comment:
```suggestion
	simulateCmd.Flags().BoolVar(&simulateReturnPCTrace, "return-pc-trace", false, "Return PC trace of app calls during simulation")
```

Reply:
see [pr response on clerk and oas2](https://github.com/algorand/go-algorand/pull/5360/commits/02cc01a00aff1061e7ff90651b5269e637309f8a)